src/components/Answers/__tests__/Answers-test.tsx

```diff
@@ -0,0 +1,92 @@
+/** @jsx h */
+
+import { h } from 'preact';
+import { render } from '@testing-library/preact';
+import Answers, { AnswersProps } from '../Answers';
+
+const defaultProps: AnswersProps = {
+  hits: [],
+  isLoading: false,
+  cssClasses: {
+    root: 'root',
+    header: 'header',
+    emptyRoot: 'empty',
+    loader: 'loader',
+    list: 'list',
+    item: 'item',
+  },
+  templateProps: {
+    templates: {
+      header: 'header',
+      loader: 'loader',
+      item: 'item',
+    },
+  },
+};
+
+describe('Answers', () => {
+  describe('Rendering', () => {
+    it('renders without anything', () => {
+      const { container } = render(<Answers {...defaultProps} />);
+      expect(container.querySelector('.root')!.classList).toContain('empty');
+      expect(container).toMatchSnapshot();
```

Comment:
How about inlining snapshots? It's easy to lose context and miss parts when reviewing this kind of snapshots.